exercise_060_cluster_base/README.md

```diff
@@ -0,0 +1,37 @@
+# Demonstrate Akka Cluster Weakly-Up members
+
+This exercise is a duplicate of the `cluster_weakly_up` exercise but implemented
+in Akka Typed.
```

Comment:
I wonder if I wouldn't just replace the other one ;). We like to call Akka Typed simply "the new Actor API's" these days (and untyped is "Classic")
```suggestion
using the new typed Actor API's.
```

Reply:
The plan is to replace all exercises that use the Akka Classic API's with the Akka Typed API's. The approach would be to add the Akka Typed version in a new series (like starting at exercise 60) and leave the two versions of each exercise in the repo. After some time (TBD), we'd remove the 'old' version and move the 'new' version to the former exercise number range.